### DISCUSSION TITLE
How to uninstall

### DISCUSSION THREAD

**Discussion by namuhamai** [winglessraven/AMP-Discord-Bot#121] [Q&A] [answered]
How can I uninstall AMP Discord Bot from my AMP instance?
Do I just need to remove certain files and modify certain files?
If so, which files need to be removed, and which files need to be modified?

**Comment by winglessraven** [comment, marked as answer]
Essentially just do a reverse of the manual install steps.

1) Stop the instance
2) Edit the AMPConfig.conf file to remove `DiscordBotPlugin` where it was added
3) Remove the DiscordBotPlugin folder and dll file
4) Start the instance again

**Reply by namuhamai** [reply]
Thank you!
